client/rest/src/index.js

```diff
@@ -122,10 +59,7 @@ const registerRoutes = (server, db, services) => {
 	const servicesView = {
 		config: {
 			network: services.config.network,
-			rest: {
-				protocol: services.config.protocol,
-				port: services.config.port
-			},
+			restEndpoint: `${services.config.protocol}://localhost:${services.config.port}`,
```

Comment:
Should this be restEndpoint in the config?

Reply:
this is only when rosetta is hosted within main REST, so, this is just endpoint of REST (which is shared with rosetta REST)